### PR TITLE
Add user-specified field for "Other" legal status

### PR DESCRIPTION
**Pull request by JeremyKennedy**
Adds the field to the backend. It is required when legal status is Other.

Currently we are not doing anything else with this field.

- [x] Frontend support for the `string` field type @KhalidAdan

### CODE REVIEW

**Comment by KhalidAdan** [comment]
This all looks good! Ping me again when it's not in draft for the approval!

**Comment by JeremyKennedy** [comment]
tbh don't 100% recall how I left this, but I think I left it in draft because it needs support on your end before merging in!

**Review by KhalidAdan — APPROVED** [review]

**Comment by JeremyKennedy** [comment]
Reviewed, discussed, and agreed on the following:
- [x] Treat like a regular question, with a text label and a single-line text box @KhalidAdan 

Ready to merge once above is complete!

**Comment by JeremyKennedy** [comment]
Love it! Thank you!

**Pull request merged**
